### ISSUE TITLE
Tensorflow-GPU 2.0 Docker Python script is just returning a message without executing

### ISSUE THREAD

**Issue by vadimen** [tensorflow/tensorflow#36758]
I installed "tensorflow:2.0.1-gpu-py3" docker image on my Ubuntu 18.04 with nvidia-driver-435 and mx110 video card. It works and even runs the following code after few minutes of initialization
`python -c "import tensorflow as tf;print(tf.reduce_sum(tf.random.normal([1000, 1000])))"`
but when I'm trying to execute this file 

https://github.com/vadimen/algorithms/blob/master/code.py

it just returns 

`python3 ssd_mobilenet_tf_for_video.py 
2020-02-14 17:56:28.801358: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcuda.so.1
2020-02-14 17:56:28.803092: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:1006] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2020-02-14 17:56:28.803374: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1618] Found device 0 with properties: 
name: GeForce MX110 major: 5 minor: 0 memoryClockRate(GHz): 1.006
pciBusID: 0000:01:00.0
2020-02-14 17:56:28.803547: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudart.so.10.0
2020-02-14 17:56:28.804471: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcublas.so.10.0
2020-02-14 17:56:28.805256: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcufft.so.10.0
2020-02-14 17:56:28.805476: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcurand.so.10.0
2020-02-14 17:56:28.806716: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcusolver.so.10.0
2020-02-14 17:56:28.807675: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcusparse.so.10.0
2020-02-14 17:56:28.810783: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudnn.so.7
2020-02-14 17:56:28.810947: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:1006] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2020-02-14 17:56:28.811578: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:1006] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2020-02-14 17:56:28.812142: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1746] Adding visible gpu devices: 0
2020-02-14 17:56:30.518085: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA
2020-02-14 17:56:30.540075: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 1800000000 Hz
2020-02-14 17:56:30.540367: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x4f31d60 executing computations on platform Host. Devices:
2020-02-14 17:56:30.540380: I tensorflow/compiler/xla/service/service.cc:175]   StreamExecutor device (0): Host, Default Version
2020-02-14 17:56:30.577626: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:1006] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2020-02-14 17:56:30.577961: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x4eda270 executing computations on platform CUDA. Devices:
2020-02-14 17:56:30.577977: I tensorflow/compiler/xla/service/service.cc:175]   StreamExecutor device (0): GeForce MX110, Compute Capability 5.0
2020-02-14 17:56:30.578121: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:1006] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2020-02-14 17:56:30.578356: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1618] Found device 0 with properties: 
name: GeForce MX110 major: 5 minor: 0 memoryClockRate(GHz): 1.006
pciBusID: 0000:01:00.0
2020-02-14 17:56:30.578399: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudart.so.10.0
2020-02-14 17:56:30.578417: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcublas.so.10.0
2020-02-14 17:56:30.578431: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcufft.so.10.0
2020-02-14 17:56:30.578446: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcurand.so.10.0
2020-02-14 17:56:30.578462: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcusolver.so.10.0
2020-02-14 17:56:30.578477: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcusparse.so.10.0
2020-02-14 17:56:30.578489: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudnn.so.7
2020-02-14 17:56:30.578537: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:1006] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2020-02-14 17:56:30.578769: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:1006] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2020-02-14 17:56:30.578972: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1746] Adding visible gpu devices: 0
2020-02-14 17:56:30.579001: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudart.so.10.0
2020-02-14 17:56:30.579466: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1159] Device interconnect StreamExecutor with strength 1 edge matrix:
2020-02-14 17:56:30.579477: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1165]      0 
2020-02-14 17:56:30.579483: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1178] 0:   N 
2020-02-14 17:56:30.579558: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:1006] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2020-02-14 17:56:30.579815: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:1006] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2020-02-14 17:56:30.580074: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1304] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 1535 MB memory) -> physical GPU (device: 0, name: GeForce MX110, pci bus id: 0000:01:00.0, compute capability: 5.0)
{'detection_classes': TensorShape([None, 100]), 'num_detections': TensorShape([None]), 'detection_boxes': TensorShape([None, 100, 4]), 'detection_scores': TensorShape([None, 100])}
`

**Comment by gowthamkpr** [comment]
@vadimen Can you plese check using the latest version of tensorflow-gpu docker image and let me know if it works. Thanks!

**Comment by vadimen** [comment]
@gowthamkpr I installed tensorflow/tensorflow:nightly-gpu-py3 image released 8 hours ago and everything did work. I couldn't get GUI output from opencv but it's clear why, at least I got console output from python which told me that my code is executed 5 times slower than on CPU on host machine. I think it's because of not enough memory, my mx110 card gas 2GB of memory but output says it's all used 

2020-02-20 10:48:00.273011: W tensorflow/core/common_runtime/bfc_allocator.cc:245] Allocator (GPU_0_bfc) ran out of memory trying to allocate 2.27GiB with freed_by_count=0. The caller indicates that this is not a failure, but may mean that there could be performance gains if more memory were available.

why is all memory being used ? 
anyway the initial issue seems to be solved. Thank you so much for that.
